rescript.md

```diff
@@ -9,7 +9,7 @@
 7. All new components should be Jsx3 components. This is done to be able to use the latest language features.
 8. When deciding between `thing->function->function` and `function(thing)->function` read it out and pick the one that makes more sense in a natural language, i.e. `Firebase.firestore(firebase)->Firebase.root("main")` and `maybeSomething->Belt.Option.getWithDefault("definitelySomething")`
 9. Liberally simplify functions from other modules at the top of your code. Eg prefer `a->mul(b)` to `a->Ethers.BigNumber.mul(b)` by declaring it `let {mul} = module(Ethers.BigNumber)`. Re-name them to avoid collisions.
-10. When defining functions keep the types definition in the function, not in the variable, to allow more freedom for type inference, i.e. `let a = (b: string, c: int): string => b === "ok"` over `let a: ((string, int) => boolean) = (b, c) => b === "ok"`
+10. When defining functions keep the types definition in the function, not in the variable, to allow more freedom for type inference, i.e. `let a = (b: string, c: int): string => b === "ok"` over `let a: ((string, int) => boolean) = (b, c) => b === "ok"` [What's the difference between these two for the compiler?]
```

Comment:
also, should the first one be `let a = (b: string, c: int): boolean => b === "ok"` as opposed to `let a = (b: string, c: int): string => b === "ok"`?

Reply:
Yes, the first one is definitely a boolean, sorry about that.

I'm not sure super sure about how the compiler treats this one differently, it is copied from the template that I used as a reference. Maybe it is worth removing, I don't think it is actually a real concern.